### ISSUE TITLE
`make setup` fails if python 3.7 is not installed

### ISSUE THREAD

**Issue by chrysle** [mkdocstrings/mkdocstrings#576]
**Describe the bug**
When I run `make setup` to setup the `mkdocstrings` development environment, I get the following error:

```
......
[NoPythonVersion]: No Python interpreter matching python3.7 is found.
make: *** [Makefile:38: setup] Error 1
```

**To Reproduce**

Run `make setup` with missing python interpreter in version 3.7.

**Expected behavior**

`make setup` would succeed without having to install Python 3.7.

**Screenshots**

**Information (please complete the following information):**
- OS: Ubuntu 20.04.05 LTS x86_64
- Browser: -
- `mkdocstrings` version: Development (current tip of master branch) 

**Additional context**

This line in the setup script:

https://github.com/mkdocstrings/mkdocstrings/blob/f9799b92cb25dd4e02b6937bbfb438bfc858f889/scripts/setup.sh#L17

seems to require that Python in versions 3.7+ is installed. I suggest changing the behavior.

**Comment by pawamoy** [comment]
Thanks for the report. I'll add an option to the multirun plugin to avoid failure when an interpreter isn't found :+1:

**Comment by pawamoy** [comment]
OK I've released pdm-multirun 0.3.0 which by default will continue even when an interpreter isn't found. The new version adds a `--fail-fast` flag to the multirun command to exit early instead of continuing with other interpreters.
